examples/spring-boot-webflux/src/test/resources/application-testbed.yml

```diff
@@ -1,3 +1,8 @@
+# Use-legacy-processing to use this application-testbed.yml file instead of application.yml
+spring:
+  config:
+    use-legacy-processing: true
```

Comment:
Can we do the same for other examples, rather than deleting `application-testbed.yml`?